.github/workflows/unit-tests-gh-hosted.yml

```diff
@@ -46,7 +49,7 @@ jobs:
       - name: Start MongoDB
         uses: supercharge/mongodb-github-action@1.7.0
         with:
-          mongodb-version: 4.4
+          mongodb-version: ${{ matrix.mongo-version}}
```

Comment:
Could you please open a temporary PR against our fork so that we can verify this workflow?

Reply:
done. here's the PR https://github.com/open-craft/edx-platform/pull/639